cypress/integration/console/shared/collaborators/edit.spec.js

```diff
@@ -0,0 +1,333 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Collaborators', () => {
+  const userId = 'main-collab-user'
+  const user = {
+    ids: { user_id: userId },
+    primary_email_address: 'main-collab-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+  const collaboratorId = 'collab-test-user'
+  const collaboratorUser = {
+    ids: { user_id: collaboratorId },
+    primary_email_address: 'collab-test-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+  const organizationId = 'test-collab-org'
+  const organization = {
+    ids: { organization_id: organizationId },
+  }
+
+  const orgCollaborator = {
+    collaborator: {
+      ids: {
+        organization_ids: {
+          organization_id: organizationId,
+        },
+      },
+      rights: ['RIGHT_APPLICATION_ALL'],
```

Comment:
Please adjust user/org rights for each entity. e.g. use `RIGHT_GATEWAY_ALL` for gateway collaborators test. same for other entities.